src/sentry/api/serializers/models/project.py

```diff
@@ -983,19 +942,25 @@ def get_value_with_default(key):
                 "verifySSL": bool(attrs["options"].get("sentry:verify_ssl", False)),
                 "scrubIPAddresses": bool(attrs["options"].get("sentry:scrub_ip_address", False)),
                 "scrapeJavaScript": bool(attrs["options"].get("sentry:scrape_javascript", True)),
-                "groupingConfig": get_value_with_default("sentry:grouping_config"),
-                "groupingEnhancements": get_value_with_default("sentry:grouping_enhancements"),
-                "groupingEnhancementsBase": get_value_with_default(
-                    "sentry:grouping_enhancements_base"
+                "groupingConfig": self.get_value_with_default(attrs, "sentry:grouping_config"),
```

Comment:
what's the difference between calling `self.` vs using the method itself?

Reply:
if a method is part of a class, you have to use self to access it. (similar to `this` in javascript)

Reply:
oh i see, the `get_value_with_defaults` function got moved out of the serializer -- what was the reasoning for that?

Reply:
it got moved into the serializer, and out of the function (in python you can define functions in functions that have the scope of the outer function)